packages/react-scripts/proxy/proxies.js

```diff
@@ -19,32 +19,35 @@ const proxies = [
     accept: 'application/' /* 'application/x-gedcomx-v1+json', 'application/json', etc. */
   },
   {
-    route: '/ask',
+    route: '/ask'
```

Comment:
did your editor make all these changes?

Reply:
No, I did.

Reply:
this is our style, so please don't change it without a good reason. It will just get put back when one of our editors touches it again.